packages/editor/src/components/editor-help/test/index.native.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import { fireEvent, render, waitForElementToBeRemoved } from 'test/helpers';
+
+/**
+ * Internal dependencies
+ */
+import EditorHelp from '../index';
+
+it( 'lists help topics', async () => {
+	const screen = render( <EditorHelp isVisible /> );
+	const helpTopic = await screen.findByText( 'Customize blocks' );
+
+	expect( helpTopic ).toBeTruthy();
+} );
+
+it( 'navigates to help topic detail screen', async () => {
+	const screen = render( <EditorHelp isVisible /> );
+	const helpTopic = await screen.findByText( 'Customize blocks' );
+	fireEvent.press( helpTopic );
+
+	expect(
+		screen.getByText(
+			'Each block has its own settings. To find them, tap on a block. Its settings will appear on the toolbar at the bottom of the screen.'
+		)
+	).toBeTruthy();
+} );
+
+it( 'navigates back from help topic detail screen', async () => {
+	const screen = render( <EditorHelp isVisible /> );
+	const helpTopic = await screen.findByText( 'Customize blocks' );
+	fireEvent.press( helpTopic );
+
+	const detailScreenTitle1 = screen.getByText(
+		'Each block has its own settings. To find them, tap on a block. Its settings will appear on the toolbar at the bottom of the screen.'
+	);
+	expect( detailScreenTitle1 ).toBeTruthy();
+
+	const backButton = screen.getAllByA11yLabel( 'Go back' );
+	fireEvent.press( backButton[ backButton.length - 1 ] );
+
+	// Currently logs `act` warning due to https://git.io/JYYGE
+	await waitForElementToBeRemoved( () =>
+		screen.getByText(
+			'Each block has its own settings. To find them, tap on a block. Its settings will appear on the toolbar at the bottom of the screen.'
+		)
+	);
+} );
+
+it( 'dismisses when close button is pressed', async () => {
+	const closeMock = jest.fn();
+	const screen = render( <EditorHelp isVisible close={ closeMock } /> );
+	const closeButton = await screen.findByA11yLabel( 'Go back' );
+	fireEvent.press( closeButton );
+
+	expect( closeMock ).toHaveBeenCalled();
```

Comment:
I was wondering if we should also verify that the modal is actually dismissed but I realized that the `BottomSheet` component is always rendered, so there's no easy way to determine this unless we check the `isVisible` prop.

Reply:
> Also, I might recommend that we test that specific functionality with a test targeting BottomSheet instead. BottomSheet already is already fairly complex, so I believe it likely warrants its own tests.

Totally agreed, `BottomSheet` component is big and complex enough to deserve its own tests.